template_lib.py

```diff
@@ -88,6 +89,19 @@ def _render_cmd(code: str) -> str:
 
 
 class Renderer(markdown_code_blocks.CodeRenderer):
+    def link(
+        self, link: str, text: Optional[str], title: Optional[str],
+    ) -> str:
+        match = NAMED_ANCHOR_RE.search(link)
+        if match:
```

Comment:
can probably just use `.startswith('_#')`

Reply:
Ah yes, at one point the regex was going to be more complicated but no longer, have replaced with that